source/03-components/modal/modal.twig

```diff
@@ -0,0 +1,23 @@
+{% set classes = [
+  'c-modal',
+  'js-modal',
+  modifier_classes ? modifier_classes : '',
+]|join(' ')|trim %}
+
+<dialog
+  id="modal-{{ id }}"
+  {{ add_attributes({ 'class': classes })}}
+  {{ default_open ? 'data-default-open' : '' }}
+  aria-labelledby="modal-title-{{ id }}"
+  aria-describedby="modal-content-{{ id }}"
+>
+  <div class="c-modal__inner js-modal-inner">
+    <button class="c-modal__close-button js-modal-close" aria-label="Close modal" aria-controls="modal-{{ id }}" autofocus>Close Modal</button>
```

Comment:
The text (and case) should be the same for button text and `aria-label` and they should be translatable:

`{{ 'Close modal'|t }}`

Reply:
Button text removed.